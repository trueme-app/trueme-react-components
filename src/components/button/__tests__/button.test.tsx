import system from '@trueme/design-system'
import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../index'

describe('Button Component', () => {
  test('Button renders correctly', () => {
    const component = renderer.create(<Button variant='primary' theme={system}>This is a button</Button>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
