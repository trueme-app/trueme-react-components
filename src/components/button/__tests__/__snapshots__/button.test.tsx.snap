// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Button renders correctly 1`] = `
<button
  className="sc-bdVaJa lcgZoM"
  disabled={false}
  reversed={false}
  shape="pill"
  size="normal"
>
  This is a button
</button>
`;
